vyper/optimizer.py

```diff
@@ -139,14 +144,14 @@ def optimize(node: LLLnode) -> LLLnode:
             valency=node.valency,
         )
     elif node.value == "clamp" and int_at(argz, 0) and int_at(argz, 1) and int_at(argz, 2):
-        if get_int_at(argz, 0, True) > get_int_at(argz, 1, True):
+        if get_int_at(argz, 0, True) > get_int_at(argz, 1, True):  # type: ignore
```

Comment:
why ignore the type?

Reply:
mypy is seriously dumb here; using the suggested hacks like above made the code look really messy.